sdk/python/jobs/pipelines/2b_train_cifar_10_with_pytorch/train-model/main.py

```diff
@@ -15,9 +15,11 @@
 import torch.nn as nn
 import torch.nn.functional as F
 import torch.optim as optim
+import numpy as np
+import copy
```

Comment:
nit: let's redo imports as 
```
import argparse
import copy
import os

import mlflow
import numpy as np
import torch
import torchvision
import torchvision.transforms as transforms
import torch.nn as nn
import torch.nn.functional as F
import torch.optim as optim
```
this way system imports are at the top + imports are alphabetized